pkg/tests/ossm/smoke_test.go

```diff
@@ -61,33 +61,21 @@ func TestSmoke(t *testing.T) {
 		t.LogStep("Install bookinfo pods and sleep pod")
 		app.InstallAndWaitReady(t, app.Bookinfo(ns), app.SleepNoSidecar(ns))
 
-		t.NewSubTest(fmt.Sprintf("install bookinfo with smcp %s", fromVersion)).Run(func(t TestHelper) {
+		t.LogStepf("Install SMCP %s and verify it becomes ready", fromVersion)
+		assertSMCPDeploysAndIsReady(t, fromVersion)
 
-			t.LogStepf("Create SMCP %s and verify it becomes ready", fromVersion)
-			assertSMCPDeploysAndIsReady(t, fromVersion)
-			if env.GetSMCPVersion().GreaterThan(version.SMCP_2_2) {
-				assertRoutesExist(t)
-			}
-
-			t.LogStep("Restart all pods to verify proxy is injected in all pods of Bookinfo")
-			oc.RestartAllPods(t, ns)
-			retry.UntilSuccess(t, func(t test.TestHelper) {
-				assertSidecarInjectedInAllBookinfoPods(t, ns)
-			})
-
-			t.LogStep("Check if bookinfo productpage is running through the Proxy")
-			assertTrafficFlowsThroughProxy(t, ns)
-
-			t.LogStep("verify proxy startup time. Expected to be less than 10 seconds")
-			t.Log("Jira related: https://issues.redhat.com/browse/OSSM-3586")
-			t.Log("From proxy json , verify the time between status.containerStatuses.state.running.startedAt and status.conditions[type=Ready].lastTransitionTime")
-			t.Log("The proxy startup time should be less than 10 seconds for ratings pod")
-			assertProxiesReadyInLessThan10Seconds(t, ns)
+		t.LogStep("Restart all pods to verify proxy is injected in all pods of Bookinfo")
+		oc.RestartAllPods(t, ns)
+		retry.UntilSuccess(t, func(t test.TestHelper) {
+			assertSidecarInjectedInAllBookinfoPods(t, ns)
 		})
```

Comment:
Why install bookinfo first, then install smcp, and then restart bookinfo? Why not simply install smcp first and then install bookinfo?

Reply:
We talked about this in some point, we agree to install first, verify that is working without istio and then validate again that works also after istio proxy are up. I can move this and do all with the installation after the smcp install

Reply:
Changed